tests/framework/extensions/machinepools/machinepools.go

```diff
@@ -61,10 +64,13 @@ func NewRKEMachinePool(controlPlaneRole, etcdRole, workerRole bool, poolName str
 // 	"worker":       false,
 // },
 // }
-func RKEMachinePoolSetup(nodeRoles []map[string]bool, machineConfig *unstructured.Unstructured) []apisV1.RKEMachinePool {
-	machinePools := []apisV1.RKEMachinePool{}
+func RKEMachinePoolSetup(nodeRoles []map[string]bool, hasWindows bool, machineConfig *unstructured.Unstructured) []apisV1.RKEMachinePool {
+	var machinePools []apisV1.RKEMachinePool
 	for index, roles := range nodeRoles {
-		machinePool := NewRKEMachinePool(roles["controlplane"], roles["etcd"], roles["worker"], "pool"+strconv.Itoa(index), 1, machineConfig)
+		machinePool := NewRKEMachinePool(roles["controlplane"], roles["etcd"], roles["linuxWorker"], "pool"+strconv.Itoa(index), rke2.DefaultMachineOS, 1, machineConfig)
```

Comment:
When if we're change `worker` to `linuxWorker` we need to update this in our existing tests. Is it super necessary to rename this?

Reply:
that is a typo, I was testing out a different method and realized it would not work without a massive rewrite that ultimately made no sense. I will get that fixed.